apps/src/templates/sectionProgressV2/ProgressIcon.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import {ITEM_TYPE, ITEM_TYPE_SHAPE} from './ItemType';
+import styles from './progress-table-legend.module.scss';
+import FontAwesome from '../FontAwesome';
+import ProgressBox from '../sectionProgress/ProgressBox';
+import classNames from 'classnames';
+
+export default function ProgressIcon({itemType}) {
+  const needsFeedbackTriangle = () => (
+    <div className={classNames(styles.needsFeedback, styles.cornerBox)} />
+  );
+
+  const feedbackGivenTriangle = () => (
+    <div className={classNames(styles.feedbackGiven, styles.cornerBox)} />
+  );
+
+  const notStartedBox = () => (
+    <ProgressBox
+      started={false}
+      incomplete={20}
+      imperfect={0}
+      perfect={0}
+      lessonIsAllAssessment={false}
+    />
+  );
+
+  const viewedBox = () => (
+    <ProgressBox
+      started={false}
+      incomplete={20}
+      imperfect={0}
+      perfect={0}
+      lessonIsAllAssessment={false}
+      viewed={true}
+    />
+  );
+
+  return (
+    <>
+      {itemType?.length && (
+        <FontAwesome
+          id={'uitest-' + itemType[0]}
+          icon={itemType[0]}
+          style={{color: itemType[1]}}
+          className={styles.fontAwesomeIcon}
+        />
+      )}
+      {itemType === ITEM_TYPE.NOT_STARTED && notStartedBox()}
+      {itemType === ITEM_TYPE.VIEWED && viewedBox()}
```

Comment:
You are probably already tracking, but wanted to call out we will eventually need to have multiple states. E.g. a completed level that needs feedback.

Reply:
The `NEEDS_FEEDBACK` "icon" showing the upper and right borders of the adjoining grid square is only used in the legend. In the actual table grid we'd show the icon for the completion status (if any) and also apply the CSS class to put the triangle in the corner of the box.